R/getLargeScaleCharacteristics.R

```diff
@@ -269,7 +269,8 @@ getLargeScaleCharacteristics <- function(cdm,
     study_table <- cdm[[table_name]] %>%
       dplyr::inner_join(subjects, by = "person_id") %>%
       # rename start date
-      dplyr::rename("start_date" = .env$start_date)
+      dplyr::rename("start_date" = .env$start_date) %>%
+      dplyr::mutate("concept_name" = .env$concept_id)
```

Comment:
Okay now I understand your question to obtain concept name you must do:
```
study_table %>%
  dplyr::left_join(
    cdm$concept %>%
      dplyr::select("concept_id", "concept_name"),
    by = "concept_id"
  )
```

Reply:
ok thank you, I think it's done. Please also kindly check issues 5,6 are implemented the way we want.